judge/views/api/api_v2.py

```diff
@@ -315,6 +315,8 @@ def get_object_data(self, participation):
         return {
             'user': participation.user.username,
             'contest': participation.contest.key,
+            'start_time': participation.start.isoformat(),
```

Comment:
The fields these properties access will need to be added to the `.only()`

- `contest.start_time`
- `contest.end_time`
- `contest.time_limit`
- `real_start`